changelog.md

```diff
@@ -3,5 +3,7 @@
 ## changelog for version 1.0.13
 
 ### Fix
+- Updated line endings
 - Added validation for required settings
-- Error saving card should not cause transaction to fail
\ No newline at end of file
+- Error saving card should not cause transaction to fail
+- Validate payment info
```

Comment:
Were these added to v13 or v14?

Reply:
These were added to version 13
Updating the version now after release for all new features.
New changes will have their own entry in the changelog
This text is also used in the WordPress store, do you think we should change it here? I don't know if it can be changed on the store.

Reply:
Try changing it there as well.
Shouldn't you also write what changed for v14? Even if only that you upgraded the WP version

Reply:
Nothing changed (yet) for v14
I'm confused what should I change

Reply:
We should just have release noted for v14 saying that we updated the WordPress version

Reply:
Oh, the wordpress version was updated in v13

Reply:
So that new changes are on the new version

Reply:
Makes more sense to move to the next version for development after each release